src/local_scheduler/local_scheduler.cc

```diff
@@ -246,6 +248,19 @@ void start_worker(LocalSchedulerState *state, ActorID actor_id) {
   start_actor_worker_command[num_args + 1] = (const char *) id_string;
   start_actor_worker_command[num_args + 2] = NULL;
   /* Try to execute the worker command. Exit if we're not successful. */
+
+  char worker_id[ID_STRING_SIZE];
+  ObjectID_to_string(globally_unique_id(), worker_id, ID_STRING_SIZE);
+
+  std::string outfile = std::string("/tmp/raylogs/worker:") +
+                      std::string(worker_id) + std::string(".out");
+  std::string errfile = std::string("/tmp/raylogs/worker:") +
+                      std::string(worker_id) + std::string(".err");
+
+  int fd_out = open(outfile.c_str(), O_WRONLY | O_CREAT, S_IRUSR | S_IWUSR);
```

Comment:
Let's add a comment documenting that we're redirecting the worker process's STDOUT and STDERR to newly created files so that they aren't mixed with the local scheduler's STDOUT and STDERR.